torch_xla_py/xla_model.py

```diff
@@ -140,8 +140,10 @@ def xla_replication_devices(local_devices):
 
 class RateTracker(object):
 
-  def __init__(self, smooth_factor=0.8):
-    self._smooth_factor = smooth_factor
+  def __init__(self, smooth_factor=None):
+    self._smooth_factor = xu.getenv_as(
+        'RATE_TRACKER_SMOOTHING', float,
+        0.8) if smooth_factor is None else smooth_factor
```

Comment:
Isn't this shorter?
```Python
def __init__(self, smooth_factor=xu.getenv_as('RATE_TRACKER_SMOOTHING', float, 0.8)):
```

Reply:
I thought that goes against some pep8 convention (don't call function in default value assignment)

Reply:
SG then!